contributors/triage/guideline.md

```diff
@@ -0,0 +1,60 @@
+
+## Validate if the issue is real
+
+Validate if the problem is real by reproducing it. If reproducible, move to the next step of finding a right SIG. You may need to contact the issue reporter in the following cases:
+* Do a quick duplicate search to see if the issue has been reported already. If a duplicate is found, let the issue reporter know it by marking it duplicate.
+* If you can not reproduce the issue, contact the issue reporter with your findings and close the issue if both the parties agree that it could not be reproduced.
+* If the issue is non-trivial to reproduce, work with issue reporter or move to next step of finding a right SIG, and let SIG know of your findings.
+* If you do not get a response in 15 days, then close the issue with appropriate comment.
+
+## Find the right SIG(s)
+Components are divided among [Special Interest Groups (SIGs)](https://github.com/kubernetes/community/blob/master/sig-list.md). Find a proper SIG for the ownership of the issue using the bot:
+
+* Typing `/sig network` in a comment should add the sig/network label, for example.
+* Multiword SIGs use dashes, for example `/sig cluster-lifecycle`.
+
+## Define priority
+After you validate the issue, add a comment of what you think of priority of fixing issue - critical, high, medium, low, wishlist. The owner from appropriate SIG will assign a correct priority.
+* `priority/failing-test` or `priority/critical-urgent` (P0 - Critical - fix it now, it’s a blocker)
+* `priority/important-soon` or `priority/important-longterm`  (P1 - High - functional error requiring quick fix)
+* `priority/backlog`  (P2 - Medium - error but not critical, small issue, for example, typo)
+* `priority/awaiting-more-evidence` (P3 - Low)
+* (P4 - Wishlist - not a real bug but suggestion or new feature request)
+
+## Set ownership
+
+All issues should have an owner. If you are not sure of an owner, defer to the SIG label only. You can ping a team with an @ mention, like `@kubernetes/sig-cluster-lifecycle, can you have a look at this?` if you feel the issue should warrant a notification.
+
+If you think you can fix the issue, assign it to yourself with just `/assign`.
+
+## Poke issue owner if PR is not created for it in 30 days
+
+If you see any issue which is owned by a developer but a PR is not created in 30 days, a Triage engineer should contact the issue owner and ask for PR or release ownership.
+
+## Poke SIG if a SIG label is assigned to the issue but no one took ownership of it in 15 days
```

Comment:
This section is directory to the SIG - i.e. the issue has sig label but no response from SIG in certain numbers of days so poke/gently remind the SIG itself. As you said, it would be nice and helpful if we can have specific owner - a SIG lead, may be? Thanks!